packages/react-components/react-progress/src/components/Progress/Progress.types.ts

```diff
@@ -1,17 +1,52 @@
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import { Label } from '@fluentui/react-label';
 
 export type ProgressSlots = {
-  root: Slot<'div'>;
+  /**
+   * The root of the Progress
+   * The root slot receives the `className` and `style` specified directly on the `<Progress>`.
+   */
+  root: NonNullable<Slot<'div'>>;
+  /**
+   * The title of the Progress.
+   * The label slot receives the styling related to the title associated with the Progress.
+   */
+  label?: Slot<typeof Label>;
+  /**
+   * The animated slot of the Progress
+   * The indicator slot receives the styling related to the loading bar associated with the Progress
+   */
+  indicator?: Slot<'span'>;
```

Comment:
Consider Slot<'progress'> or defaulting the type in useProgress to output a <progress>

Reply:
I used a span because it was easier to style a div than to override the default styling of the default HTML progress element